README.md

```diff
@@ -40,18 +55,6 @@ Output example
 
 ![Example output](https://user-images.githubusercontent.com/3704611/53132247-845f7080-356f-11e9-9c76-a9498d4a744b.png)
 
-Why even care about unsafe Rust usage?
---------------------------------------
-
-When and why to use unsafe Rust is out of scope for this project; it is simply
```

Comment:
Was it intended to drop the statement "When and why to use unsafe Rust is out of scope for this project"?

Reply:
It was moved - it's redundant to point out what is not in scope or not - it's now rather generic statement without it's own heading at top.